src/js/rs-release/rs-release.js

```diff
@@ -193,35 +199,29 @@ EvaRsRelease.prototype = {
             }
         });
 
-        var table = '<table id="rs-release-table-by-assembly" class="responsive-table hover tablesorter table-fixed">' +
+        var table = '<table id="rs-release-table-by-assembly-new-data" class="responsive-table hover tablesorter table-fixed">' +
                     '<thead>' +
                     '<tr>' +
-                        '<th>Scientific name</th>' +
-                        '<th>Taxonomy ID</th>' +
+                        '<th>Download link</th>' +
                         '<th>Assembly accession</th>' +
-                        '<th><div title="RS IDs that were clustered in the current release">New Current RS <i class="icon icon-generic" data-icon="i"></div></th>' +
-                        '<th><div title="RS IDs that were issued for remapped variants">Remapped clustered <i class="icon icon-generic" data-icon="i"></div></th>' +
-                        '<th><div title="RS IDs that were created for new variants">Newly clustered <i class="icon icon-generic" data-icon="i"></div></th>' +
-                        '<th><div title="RS IDs that should NOT be used because they are merged into another active RS">Merged RS <i class="icon icon-generic" data-icon="i"></div></th>' +
-                        '<th><div title="RS IDs that were split into multiple active RS due to remapping">Split RS <i class="icon icon-generic" data-icon="i"></div></th>' +
+                        '<th><div title="RS IDs that were added in the current release">New Current RS<i class="icon icon-generic" data-icon="i"></div></th>' +
```

Comment:
This is called "New Current RS" in this table but "New RS IDs" in the per-species new data table... should probably be consistent.